Prototyper/Classes/Feedback/FeedbackView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FeedbackView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 20.12.19.
+//
+
+import SwiftUI
+
+struct FeedbackView: View {
+    @EnvironmentObject var model: Model
+    @State var descriptionText: String = ""
+    @State var showSheet: Bool = false
+    @State var showScreenshot: Bool = true
+    @State var activeSheet: ActiveSheet = .loginSheet
+    @State var showSendFeedbackView: Bool = false
+    @State var feedback: Feedback?
+    
+    enum ActiveSheet {
+        case loginSheet, MarkupSheet
+    }
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                HStack (alignment: .top) {
+                    if showScreenshot {
+                        ZStack (alignment: .topTrailing) {
+                            ZStack {
+                                Image(uiImage: model.screenshot)
+                                    .resizable()
+                                    .border(Color(white: 0.75))
+                                    .scaledToFit()
+                                Image(systemName: "hand.draw.fill")
+                                    .imageScale(.large)
+                                    .onTapGesture {
+                                        self.editImage()
+                                }
+                            }
+                            Image(systemName: "xmark.circle.fill")
+                                .imageScale(.large)
+                                .onTapGesture {
+                                    self.removeScreenshot()
+                            }
+                        }
+                    }
+                    NavigationLink(destination: SendFeedbackView(showSendFeedbackView: $showSendFeedbackView, feedback: $feedback), isActive: $showSendFeedbackView) {
+                        Text("")
+                    }
+                    MultilineTextView(text: $descriptionText, placeholderText: "Add your feedback here...").frame(numLines: 15)
+                }
+                Spacer()
+            }.padding()
+                .navigationBarTitle("Write Feedback")
```

Comment:
I would name this the same as the action that is displayed when you press the Feedback button: "Give Feedback"